### DISCUSSION TITLE
3차 데모데이 발표자 선정

### DISCUSSION THREAD

**Discussion by Hogu59** [woowacourse-teams/2024-pengcook#157] [General]
안드쪽에서 2번 발표를 진행했다보니 이번에는 백엔드, 특히 새양을 제외하고 발표자를 선정해보면 어떨까 싶은데 월요일에 논의합시다요

**Comment by HaiSeong** [comment]
로그, 메트릭을 아토가 담당했으니 아토(@hyxrxn  )가 이번 발표에 적합하지 않을까 생각합니다.

**Comment by Hogu59** [comment]
분야별로 각자 맡은 역할에서 설명을 해야한다면 따로 설명해도 가능하긴 합니다.
다만 전체적인 발표자 1명만 먼저 정하면 좋을것 같다는 생각이 듭니다!
